convert/convert.go

```diff
@@ -78,6 +78,49 @@ func SamToBedFrag(s sam.Sam, fragLength int, reference map[string]chromInfo.Chro
 	}
 }
 
+//BedNameToWig uses bed entries from an input file to construct a Wig data structure where the Wig value is euqla to the float64-casted nameof an overlapping bed entry. Regions with no bed entries will be set to the value set by Missing (default 0 in the cmd).
+func BedNameToWig(inFile string, reference map[string]chromInfo.ChromInfo, Missing float64) []wig.Wig {
+	wigSlice := make([]wig.Wig, len(reference))
+	var line string
+	var chromIndex int
+	var midpoint int
+	var startNum, endNum int
+	var x int
+	var i int = 0
+	var doneReading bool = false
+	var current *bed.Bed
+	var currentWig wig.Wig
+	//generate Wig skeleton from reference
+	for _, v := range reference {
```

Comment:
This could be a separate function (give a slice of chroms and get back a slice of wigs)